### PR TITLE
Reporting Demo

### PR DESCRIPTION
**Pull request by TrevorAntony**
I have updated the PR
Screenshot 2024-07-24 at 16 10 51
Screenshot 2024-07-24 at 16 11 02

### CODE REVIEW

File: packages/esm-covid-app/src/home.component.tsx [hunk 1]
```diff
@@ -1,16 +1,158 @@
...
+        }}
+      >
+        <div className={styles.datePickerContainer}>
+          <Dropdown
+            id="report-dropdown"
+            titleText="Select Report"
+            label="Select a report to display"
+            items={[
+              { id: 'ecabd559-14f6-4c65-87af-1254dfdf1304', text: 'Covid-19 Report' },
+              { id: '3ffa5a53-fc65-4a1e-a434-46dbcf1c2de2', text: 'HTS Report' },
```
Comment (CynthiaKamau) [inline packages/esm-covid-app/src/home.component.tsx:106]: Lets add these uuids in the config

**Review by CynthiaKamau — COMMENTED** [review]
Thanks @TrevorAntony , please undo the changes in the `yarn.lock` and `package.json` file

File: packages/esm-covid-app/src/home.component.tsx (hunk 1 above)
Comment (TrevorAntony) [reply packages/esm-covid-app/src/home.component.tsx]: made the requested changes. kindly check new commit

**Comment by hadijahkyampeire** [comment]
Thanks @TrevorAntony,  please look at the build failures

**Review by hadijahkyampeire — COMMENTED** [review]
Just some visual feedback, 
1. Could you leave a padding left before the reports dropdown and the fetch button.
2. Green for table headers looks too much 🤔

**Comment by hadijahkyampeire** [comment]
@TrevorAntony following our sync earlier here is some feedback we talked about, let me know if you need any assistance
- Cover at least 90% of the width for the filtering section
- Try to display the button on the same line as the filters
- Remove the green background in the table header
- Use SWR to fetch the report
- Translate the column headers

**Comment by TrevorAntony** [comment]
I have updated the PR
Screenshot 2024-07-24 at 16 10 51
Screenshot 2024-07-24 at 16 11 02

**Comment by hadijahkyampeire** [comment]
@TrevorAntony let's close this

**Comment by TrevorAntony** [comment]
closed. But i wont delete the branch for now

**Pull request closed without merging**